### PR TITLE
Fixing bug #128 of samples-is

### PR DESCRIPTION
**Pull request by Kavindu-Dodan**
## Purpose
Resolves bug #128 

## Goals
Allow OIDC sample apps to be individually deployed

+ Contains improvements on how we pick carbon jks in OIDC and SAML apps
+ Introducing a module with a listener to set carbon jks for runtime via system properties

### CODE REVIEW

File: oidc-sso-sample/pickup-dispatch/src/main/resources/jks.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+keystorename=wso2carbon.jks
+keystorepassword=wso2carbon
```
Comment (vihanga-liyanage) [inline oidc-sso-sample/pickup-dispatch/src/main/resources/jks.properties]: Please add a new line at the EOF.

File: oidc-sso-sample/pickup-manager/src/main/resources/jks.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+keystorename=wso2carbon.jks
+keystorepassword=wso2carbon
```
Comment (vihanga-liyanage) [inline oidc-sso-sample/pickup-manager/src/main/resources/jks.properties]: Please add a new line at the EOF.

File: oidc-sso-sample/oidc-jks-loader/src/main/org/wso2/sample/identity/jks/JKSLoader.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package org.wso2.sample.identity.jks;
```
Comment (omindu) [inline oidc-sso-sample/oidc-jks-loader/src/main/org/wso2/sample/identity/jks/JKSLoader.java]: Add WSO2 licence header

File: oidc-sso-sample/oidc-jks-loader/src/main/org/wso2/sample/identity/jks/JKSLoader.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class JKSLoader implements ServletContextListener {
```
Comment (omindu) [inline oidc-sso-sample/oidc-jks-loader/src/main/org/wso2/sample/identity/jks/JKSLoader.java]: Add a class level doc comment

**Comment by Kavindu-Dodan** [comment]
Suggested fixes are done.

File: oidc-sso-sample/oidc-jks-loader/src/main/org/wso2/sample/identity/jks/JKSLoader.java [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+ * This will allow us to set the carbon JKS for HTTPS communication.
+ * */
+public class JKSLoader implements ServletContextListener {
+
+    private static final Logger LOGGER = Logger.getLogger(JKSLoader.class.getName());
+
+    @Override
+    public void contextInitialized(ServletContextEvent servletContextEvent) {
+        // First find jks properties
+        final InputStream jksResource = this.getClass().getClassLoader().getResourceAsStream("jks.properties");
```
Comment (Kavindu-Dodan) [inline oidc-sso-sample/oidc-jks-loader/src/main/org/wso2/sample/identity/jks/JKSLoader.java]: @Kavindu-Dodan Variable name - jksPropertyFile

**Pull request merged**